.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: python
+
+branches:
+  except:
+    - gh-pages
+
+
+before_cache:
+  - "find ${HOME}/.pip -name log -o -name __pycache__ -type d | xargs -I {} rm -rf {}"
+
+cache:
+  directories:
+    - "${HOME}/.pip"
+
+python:
+  - 3.6
```

Comment:
Stencil was deliberately targeted at Py2 (because of AWS Lambda)
There is a py3 compatible branch, but master isn't it.